recipes/xmlf90/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "xmlf90" %}
+{% set version = "1.5.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/siesta-project/libraries/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c4492596c911b668b94fee40a53453f06aef3bc41014b9e2f4b51e640ad20528
+
+build:
+  number: 0
+  skip: true  # [win]
```

```suggestion
build:
  run_exports:
    - {{ pin_subpackage('xmlf90', max_pin='x.x.x') }}
  number: 0
  skip: true  # [win]
```
You package needs a run export to ensure that downstream packages depend on a version of this package with a compatible ABI. Please adjust max_pin according to the ABI guarantees of your library.